### ISSUE TITLE
sympy 1.6 is not importable in Python 3.5.1

### ISSUE THREAD

**Issue by oscarbenjamin** [sympy/sympy#19426]
From stackoverflow:
https://stackoverflow.com/questions/61995836/importerror-while-importing-sympy

```
>>> import sympy
Traceback (most recent call last):
  File "<pyshell#4>", line 1, in <module>
    import sympy
  File "C:\Users\Alok\AppData\Local\Programs\Python\Python35\lib\site-packages\sympy\__init__.py", line 51, in <module>
    from .core import (sympify, SympifyError, cacheit, Basic, Atom,
  File "C:\Users\Alok\AppData\Local\Programs\Python\Python35\lib\site-packages\sympy\core\__init__.py", line 4, in <module>
    from .sympify import sympify, SympifyError
  File "C:\Users\Alok\AppData\Local\Programs\Python\Python35\lib\site-packages\sympy\core\sympify.py", line 3, in <module>
    from typing import Dict, Type, Callable, Any
ImportError: cannot import name 'Type'
```

`typing.Type` was added in Python 3.5.2:
https://docs.python.org/3/library/typing.html#typing.Type

**Comment by jksuom** [comment]
Didn't we discuss this earlier somewhere?

**Comment by oscarbenjamin** [comment]
#18970 is very similar.

**Comment by oscarbenjamin** [comment]
Fixed in #19620
